lib/node_modules/@stdlib/stats/base/dists/triangular/entropy/benchmark/benchmark.js

```diff
@@ -44,9 +44,9 @@ bench( pkg, function benchmark( b ) {
 	max = new Float64Array( len );
 	mode = new Float64Array( len );
 	for ( i = 0; i < len; i++ ) {
-		min[ i ] = ( randu()*10.0 );
-		max[ i ] = ( randu()*10.0 ) + min[ i ] + EPS;
-		mode[ i ] = ( ( max[ i ] - min[ i ] ) * randu() ) + min[ i ];
+		min[ i ] = uniform( 0.0, 10.0 );
+		max[ i ] = uniform( min[ i ] + EPS, 10.0 + min[ i ] + EPS );
```

Comment:
```suggestion
		max[ i ] = uniform( min[ i ] + EPS, min[ i ] + 10.0 + EPS );
```

Just to be consistent with the other benchmarks. This change applies throughout the PR.